src/test/java/core/dto/MenuNames.java

```diff
@@ -0,0 +1,20 @@
+package core.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum MenuNames {
+  DASHBOARD("dashboard"),
+  LAUNCHES("launches"),
+  FILTERS("filters"),
+  PROJECT_SETTINGS("settings");
+
+  private final String name;
+
+  @Override
+  public String toString() {
+    return name;
+  }
```

Comment:
you already have `getName()` method, why do you override `toString()`?

Reply:
Thanks, removed